mms/model_service/mxnet_model_service.py

```diff
@@ -88,17 +89,21 @@ def __init__(self, model_name, model_dir, manifest, gpu=None):
         # Load MXNet module
         epoch = 0
         try:
-            param_filename = manifest['Model']['Parameters']
-            epoch = int(param_filename[len(model_name) + 1: -len('.params')])
+            self.param_filename = manifest['Model']['Parameters']
+            epoch = int(self.param_filename[len(model_name) + 1: -len('.params')])
+        except Exception:  # pylint: disable=broad-except
+            logger.warning('Failed to parse epoch from param file (or) parameters file not found, setting epoch to 0')
+
+        try:
+            sym, arg_params, aux_params = mx.model.load_checkpoint('%s/%s' %
+                                                                   (model_dir, manifest['Model']['Symbol'][:-12]),
+                                                                   epoch)
+            self.mx_model = mx.mod.Module(symbol=sym, context=self.ctx,
+                                          data_names=data_names, label_names=None)
+            self.mx_model.bind(for_training=False, data_shapes=data_shapes)
+            self.mx_model.set_params(arg_params, aux_params, allow_missing=True, allow_extra=True)
         except Exception:  # pylint: disable=broad-except
-            logger.warning('Failed to parse epoch from param file, setting epoch to 0')
-
-        sym, arg_params, aux_params = mx.model.load_checkpoint('%s/%s' %
-                                                               (model_dir, manifest['Model']['Symbol'][:-12]), epoch)
-        self.mx_model = mx.mod.Module(symbol=sym, context=self.ctx,
-                                      data_names=data_names, label_names=None)
-        self.mx_model.bind(for_training=False, data_shapes=data_shapes)
-        self.mx_model.set_params(arg_params, aux_params, allow_missing=True, allow_extra=True)
+            logger.warning('Failed to find symbols in the Manifest. This will be treated as Imperative model')
```

Comment:
Hmm.
When we do model export, we should capture id there is symbolic model and if there is params.
That status should be captured in the manifest, and based on the manifest we either try to load symbols and params or not.